gcp/observability/opencensus.go

```diff
@@ -121,3 +121,22 @@ func startOpenCensus(config *config) error {
 
 	return nil
 }
+
+// stopOpenCensus flushes the exporter's and cleans up globals across all
+// packages if exporter was created.
+func stopOpenCensus() {
+	if exporter != nil {
+		if sdExporter, ok := exporter.(*stackdriver.Exporter); ok {
+			sdExporter.Flush()
+			sdExporter.Close()
+		}
+
+		// Call these unconditionally, doesn't matter if not registered, will be
+		// a noop if not registered.
+		trace.UnregisterExporter(exporter)
+		view.UnregisterExporter(exporter)
+
+		internal.ClearExtraDialOptions()
+		internal.ClearExtraServerOptions()
```

Comment:
Should we do this first thing?  In case any new clients are created during this call*, they would get the census stats handlers which are already nonfunctional by the time we reach here.  Generally: cleanups often happen in the opposite order of initializers.

\* - I know they're not supposed to be and the behavior in this case would be undefined; this is more of a nit / best practice.

Reply:
Done, thanks for the suggestion.